test/python/test_kinetics.py

```diff
@@ -1065,6 +1065,32 @@ def cathode_current(phi_s, phi_l, X_Li_cathode):
         ref = np.genfromtxt(self.test_data_path / "lithium-ion-battery-test.csv")
         assert np.allclose(data, ref, rtol=1e-7)
 
+    def test_interface_current(self):
+        file = "lithium_ion_battery.yaml"
+
+        # The 'elde' electrode phase is needed as a source/sink for electrons:
+        anode = ct.Solution(file,"anode")
+        elect = ct.Solution(file,"electron")
+        elyte = ct.Solution(file,"electrolyte")
+        anode_int = ct.Interface(file,"edge_anode_electrolyte",[anode,elect,elyte])
+
+        anode.X = [0.9, 0.1]
+        elyte.X = [0.4, 0.3, 0.15, 0.15]
+
+        anode.electric_potential = 0.
+        elyte.electric_potential = 3.
+
+        phases = [anode_int, anode, elect, elyte]
+
+        for p in phases:
+            productions = anode_int.get_net_production_rates(p)
```

Comment:
Again, a name like `sdot_k` or even just `net_prod_rates` would probably be more instructive.

Reply:
Agreed, variable names are now updated. I went with `net_prod_rates` here.